unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Clipboard/Tests/ClipboardTests.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using DCL.Helpers;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+public class ClipboardTests : TestsBase
+{
+    [UnitySetUp]
+    protected override IEnumerator SetUp()
+    {
+        yield break;
+    }
+
+    [UnityTearDown]
+    protected override IEnumerator TearDown()
+    {
+        yield break;
+    }
```

Comment:
I think we can safely remove the `TestsBase` inheritance here, and thus don't need the `SetUp` and `TearDown` methods.

Reply:
done 👍